sysbuild/CMakeLists.txt

```
@@ -401,6 +400,14 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
 
   if(SB_CONFIG_MATTER)
     set_config_bool(${DEFAULT_IMAGE} CONFIG_CHIP y)
+
+    if(SB_MATTER_OTA)
```

```suggestion
    if(SB_MATTER_OTA)
```
```suggestion
    if(SB_CONFIG_MATTER_OTA)
```

due to that there are two failures in Matter CI.